internal/rpcserver/interface.go

```diff
@@ -134,3 +138,300 @@ type SyncManager interface {
 	ProcessTransaction(tx *dcrutil.Tx, allowOrphans bool, rateLimit bool,
 		allowHighFees bool, tag mempool.Tag) ([]*dcrutil.Tx, error)
 }
+
+// UtxoEntry represents a utxo entry for use with the RPC server.
```

Comment:
👍 I originally left it out since it is NOT required to be safe for concurrent access (as the underlying implementation of `blockchain.UtxoEntry` is not), but I updated this to state that explicitly now.